st2debug/bin/submit-debug-conf.yaml

```diff
@@ -0,0 +1,12 @@
+log_file_paths:
```

Comment:
Are we planning on contributing this file back as a sample to stackstorm/st2? Or should it be removed? If it stays then we'll need to address this comment here as well.